### ISSUE TITLE
String Missing Space

### ISSUE THREAD

**Issue by bittin** [calamares/calamares#1965]
Partition %1 could not be decrypted with the given passphrase.Edit the partition again and give the correct passphraseor delete and create a new encrypted partition.

String #261 in Calamares is missing a space between passphrase and or

**Comment by demmm** [comment]
Are you sure this is in the Calamares code, and not an issue in the translation?  What language used/tested?  If default `en`, then it is a code issue, if not `en`, make sure to test with other languages.

**Comment by bittin** [comment]
Problem is with this string: https://www.transifex.com/calamares/calamares/translate/#sv/calamares/418851069 so more a translation issue

**Comment by adriaandegroot** [comment]
The actual source is

```
                QString description = tr( "Partition %1 could not be decrypted "
                                          "with the given passphrase."
                                          "<br/><br/>"
                                          "Edit the partition again and give the correct passphrase"
                                          "or delete and create a new encrypted partition." )
```

(in `src/modules/partition/gui/EditExistingPartitionDialog.cpp`), so there's two `<br/>` elements between the two sentences. Translation issue.)

**Comment by adriaandegroot** [comment]
Er, no: between "passphrase" and "or". Durr, Ade, read the issue description more closely.

However, it's not necessary to raise issues in GH for each translation issue; Transifex has its own issue-reporting which gets through (although sometimes, more slowly).

**Comment by adriaandegroot** [comment]
Now really fixed in 6e8d820737dea0f3e08f12b10768facef19be684

**Comment by bittin** [comment]
great thanks :)
